src/lib.rs

```diff
@@ -689,10 +717,15 @@ impl Module {
         u: &mut Unstructured,
         ty: &FuncType,
         allocs: &mut CodeBuilderAllocations,
+        allow_invalid: bool,
     ) -> Result<Code> {
         let locals = self.arbitrary_locals(u)?;
         let builder = allocs.builder(ty, &locals);
-        let instructions = builder.arbitrary(u, self)?;
+        let instructions = if allow_invalid && u.arbitrary().unwrap_or(false) {
+            Instructions::Arbitrary(u.arbitrary()?)
```

Comment:
We can do better for `Vec<u8>` than the `Vec<A: Arbitrary>` implementation does by default (can't wait for specialization...). Basically, for "raw" byte buffers, I don't think it make sense to do the `keep_going` thing that `Vec<A: Arbitrary>` does.

```rust
fn arbitrary_vec_u8(u: &mut Unstructured) -> Vec<u8> {
    let size = u.arbitrary_len::<u8>();
    u.get_bytes(size).unwrap().to_vec()
}

// ...

let instructions = if allow_invalid && u.arbitrary().unwrap_or(false) {
    Instructions::Arbitrary(arbitrary_vec_u8(u))
} else {
    // same as before...
};
```